arch/arm/core/swi_tables.ld

```diff
@@ -3,6 +3,6 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-#if LINKER_ZEPHYR_FINAL
-INCLUDE zephyr/isr_tables_swi.ld
+#if LINKER_ZEPHYR_FINAL && defined (CONFIG_ISR_TABLES_LOCAL_DECLARATION)
```

Comment:
Formatting isn't enforced on `.ld` files, but it should try to keep some consistency with the `.clang-format` for c files.

```suggestion
#if LINKER_ZEPHYR_FINAL && defined(CONFIG_ISR_TABLES_LOCAL_DECLARATION)
```

Also for the other changes.

Reply:
I also preffer the shorter form. I just aligned my own style here do linker.ld file where it is always with a space after defined.

Reply:
Ok... not always - just the nearest the place of my edition